flink-table-store-connector/src/main/java/org/apache/flink/table/store/connector/sink/StoreSinkWriter.java

```diff
@@ -52,10 +55,13 @@
         implements StatefulSinkWriter<RowData, WriterStateT>,
                 PrecommittingSinkWriter<RowData, Committable> {
 
+    private static final BinaryRowData DUMMY_KEY = new BinaryRowData(0);
```

Comment:
It's recommended to use `BinaryRowDataUtil.EMPTY_ROW` to get ride of NullPointerException when accessing the `segments` inside the `BinaryRowData`.